### PR TITLE
fix: refactor: bump kubernetes-client to 6.12.0 (#713)

### LINKED ISSUE #713: bump kuberntes-client to latest kubernetes-client (6.12.0)
dependency to #726

### PR DESCRIPTION
**Pull request by adietish**
fixes #713, #726
depends on:
* https://github.com/redhat-developer/intellij-common/pull/211
* https://github.com/redhat-developer/intellij-common/pull/220

### CODE REVIEW

**Comment by mohitsuman** [comment]
Should this be still in draft ?

**Comment by adietish** [comment]
> Should this be still in draft ?

Yes because it depends on intellij-common to be bumped to the same client and released (1.9.4)

File: gradle.properties [hunk 1]
```diff
@@ -11,5 +11,5 @@ intellijCommonVersion=1.9.4-SNAPSHOT
 telemetryPluginVersion=1.1.0.52
```
Comment (sbouchet) [inline gradle.properties:11]: update commons to latest release

**Review by sbouchet — APPROVED** [review]
LGTM. please open an issue tio fix those failling tests

**Comment by adietish** [comment]
> LGTM. please open an issue tio fix those failling tests

done in #746

**Pull request merged**